apps/explorer/project.json

```diff
@@ -51,6 +51,42 @@
         }
       }
     },
+    "serve-devnet": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "command": "env.cmd -f ./apps/explorer/.env.devnet nx run explorer:serve"
+      }
+    },
+    "serve-testnet": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "command": "env.cmd -f ./apps/explorer/.env.testnet nx run explorer:serve"
+      }
+    },
+    "serve-capsule": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "command": "env.cmd -f ./apps/explorer/.env.capsule nx run explorer:serve"
+      }
+    },
+    "serve-stagnet1": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "command": "env.cmd -f ./apps/explorer/.env.stagnet1 nx run explorer:serve"
+      }
+    },
+    "serve-stanget2": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "command": "env.cmd -f ./apps/explorer/.env.stanget2 nx run explorer:serve"
+      }
+    },
+    "serve-mainnet": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "command": "env.cmd -f ./apps/explorer/.env.mainnet nx run explorer:serve"
+      }
+    },
```

Comment:
question (blocking): can we not use the args forwarding in order to have a single command for this, with a fallback?

https://nx.dev/packages/workspace/executors/run-commands#arguments-forwarding

Reply:
i did try that out, and indeed it reduces the verbosity of the code. that way it would look something like:

```
    "serve-env": {
      "executor": "@nrwl/workspace:run-commands",
      "options": {
        "command": "env.cmd -f ./apps/explorer/.env.{args.env} nx run explorer:serve"
      }
    },
```

what i didn't like is the lack of the ability to provide a default the env arg (without any super ugly bash hacks), and the lack of ability to tap into the error messages. for example without passing in the --env flag to the above version, the error message would say `./apps/explorer/.env.undefined` doesn't exist, which is not very intuitive imho. also, wasn't really sure about the naming of the command either, serve-env seems kinda awkward.

Reply:
thinking of creating a script for this maybe?
```
"command": "node ./scripts/get-env.js {args.env} nx run ..."
```

Reply:
I'm in favor of a script if it means we can have a single 'serve' command for each project but still get nice error messages

Reply:
alright, going to do a bit of rework on this

Reply:
@mattrussell36 @dexturr , i think i managed to clean this up, added a custom nx executor to run the dev-servers, let me know what you think.

Reply:
Nice :+1: